fem/bilininteg.hpp

```diff
@@ -2654,13 +2662,21 @@ class ElasticityIntegrator : public BilinearFormIntegrator
     where v and w are the trial and test variables, respectively, and rho/u are
     given scalar/vector coefficients. {v} represents the average value of v on
     the face and [v] is the jump such that {v}=(v1+v2)/2 and [v]=(v1-v2) for the
-    face between elements 1 and 2. For boundary elements, v2=0 such that
-    alpha=1, beta=0.5 corresponds to "outflow" on the external boundary. The
-    vector coefficient, u, is assumed to be continuous across the faces and when
-    given the scalar coefficient, rho, is assumed to be discontinuous. The
-    integrator uses the upwind value of rho, rho_u, which is value from the side
-    into which the vector coefficient, u,
-    points. */
+    face between elements 1 and 2. For boundary elements, v2=0. The vector
+    coefficient, u, is assumed to be continuous across the faces and when given
+    the scalar coefficient, rho, is assumed to be discontinuous. The integrator
+    uses the upwind value of rho, rho_u, which is value from the side into which
+    the vector coefficient, u, points.
+
+    When combined with ConservativeConvectionIntegrator integrator, the
+    coefficients alpha=1.0, beta=0.5 can be used to implement the upwind flux
+    and outflow boundary conditions in conservative form.
+
+    For the non-conservative formulation of convection using
+    ConvectionIntegrator, the transpose of this form with alpha=-1.0, beta=0.5
+    can be used to implement the upwind flux, see
+    NonconservativeDGTraceIntegrator and ex9 and ex9p.
```

Comment:
This would be more useful if these new integrators were implemented in `ex9` and `ex9p`.

Reply:
Sure, I changed both examples to use the new version now.